<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>920</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GeoViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionEnablePanel"/>
    <addaction name="actionRotate90"/>
    <addaction name="actionApplyFilter"/>
    <addaction name="actionBackgroundRemoval"/>
    <addaction name="actionTrimTop"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionGpuAcceleration"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="actionRotate90">
   <property name="text">
    <string>Rotate 90</string>
   </property>
  </action>
  <action name="actionEnablePanel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Enable panel</string>
   </property>
   <property name="toolTip">
    <string>Enable panel</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionApplyFilter">
   <property name="text">
    <string>Apply filter</string>
   </property>
  </action>
  <action name="actionGpuAcceleration">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GPU Acceleration</string>
   </property>
  </action>
  <action name="actionBackgroundRemoval">
   <property name="text">
    <string>Background removal</string>
   </property>
  </action>
  <action name="actionTrimTop">
   <property name="text">
    <string>Trim top</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionUndo_2">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo_2">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
